Client/UserResearch/OnboardingUserResearch.swift

```diff
@@ -42,23 +45,62 @@ class OnboardingUserResearch {
     }
     
     // MARK: Initializer
-    init(lpVariable: LPVar? = LPVariables.showOnboardingScreen) {
+    init(lpVariable: LPVar? = LPVariables.showOnboardingScreenAB) {
         self.lpVariable = lpVariable
     }
     
     // MARK: public
     func lpVariableObserver() {
-        Leanplum.onVariablesChanged {
-            self.updatedLPVariables?(self.lpVariable)
+        // Condition: Leanplum is disabled
+        // If leanplum is not enabled then we set the value of onboarding research to true
+        // True = .variant 1 which is our default Intro View
+        // False = .variant 2 which is our new Intro View that we are A/B testing against
```

Comment:
This true/false definition is repeated a few times, I think that isn't necessary. It is documented in ` func updateValue(value: Bool) {` (which I suggest renaming see later comment), which is good, but doesn't need to be restated elsewhere.

Reply:
I think moving forward for other A/B test I should just go with string variables encapsulated behind an Enum instead of true false. Eg. Control vs Variant

Learnt the hard way 😅